src/main/java/com/blamejared/crafttweaker/impl/loot/conditions/TriState.java

```diff
@@ -0,0 +1,17 @@
+package com.blamejared.crafttweaker.impl.loot.conditions;
+
+public enum TriState {
+    TRUE(true),
+    FALSE(false),
+    UNSET(null);
+
+    private final Boolean bool;
```

Comment:
Why are you using `Boolean` instead of `boolean`?

Reply:
Because we need to account for the `TriState` to be `UNSET`, which means that we need a third value. Vanilla uses a `null` `Boolean` to represent such a state, and since this is used in the converting method, I kept the same representation.